frontend/src/app/shared/components/user-notification/user-notification.component.ts

```diff
@@ -51,8 +67,64 @@ export class UserNotificationComponent implements OnInit, OnDestroy {
         this.httpService.put('/notification', notification.id).subscribe();
     }
 
-    ngOnDestroy() {
+    readAllNotifications() {
+        this.notifications.map(notification => ({
+            ...notification,
+            isRead: true,
+        }));
```

Comment:
why do we do this in frontend and backend? it would be better to do this only in one place